conda-recipe/run_test.sh

```diff
@@ -28,3 +28,9 @@ if [ ! -z "${DAALROOT}" ]; then
     conda remove daal --force -y
     source ${DAALROOT}/env/vars.sh
 fi
+
+# if TBBROOT is specified
+if [ ! -z "${TBBROOT}" ]; then
+    conda remove tbb --force -y
```

Comment:
tbb in conda isn't it installed by default from oneapi toolkit?

Reply:
There is no conda for oneapi for now. And then this will happen - first it will be required to split build process for 2020 and 2021 version as they have separate channels and separate packages